Core/clim-core/panes/composition.lisp

```diff
@@ -1002,12 +1002,6 @@
 (defun spacing-p (pane)
   (typep pane 'spacing-pane))
 
-(defmethod initialize-instance :after ((spacing spacing-pane) &key thickness contents &allow-other-keys)
-  (declare (ignorable thickness contents))
-  (with-slots (user-width user-min-width user-max-width
-               user-height user-min-height user-max-height)
-      spacing))
-
```

Comment:
Please describe in the commit message which methods are removed. There are lots of dumb methods in the code base :)

Reply:
Done and I alse remove an exported symbol of an old function that was removed from the code.

Reply:
Sorry, I mainly meant in the first line of the commit message. Currently the commit appears as "Cleanup code" in a compact history listing which is not informative. If you want, I can rebase and fix these cosmetic things.

Reply:
Also we, usually prefix the commit message with the changed function (e.g. `formatting-table: validate keyword arguments`), file (e.g. `stream-input: untabify`) or subsystem (e.g. `clx: fix mirror class name computation`).